receiver/gitproviderreceiver/internal/scraper/githubscraper/helpers_test.go

```diff
@@ -0,0 +1,220 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package githubscraper
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+/** TODO: Fix this test
```

Comment:
please fix or leave out

Reply:
For sure, I'm taking one of the folks at our company through the process and pairing on this contribution so that more folks can begin to contribute from Liatrio to this receiver and the OTEL community. He'll be committing that fix momentarily & adding the contributor metric as well. This will get fixed on that commit.